pkg/transformers/namereference.go

```diff
@@ -106,6 +106,97 @@ func (o *nameReferenceTransformer) Transform(m resmap.ResMap) error {
 	return nil
 }
 
+// utility function to replace a simple string by the new name
```

Comment:
Add some comment for the return values, e.g. the first return value is the actual resource name, the second return value is the namespace associated with the resource

Reply:
Done in https://github.com/kubernetes-sigs/kustomize/pull/1325/commits/9b40f8ab474007a4f2358586bc82460b4bcac577